src/xdsl/xdsl_opt_main.py

```diff
@@ -13,20 +13,21 @@
 from xdsl.dialects.builtin import *
 from xdsl.dialects.cmath import *
 from xdsl.dialects.cf import *
+from xdsl.error_format import *
 from xdsl.dialects.irdl import *
 
 
 class xDSLOptMain:
     ctx: MLContext
     args: argparse.Namespace
     """
-    The argument parsers namespace which holds the parsed commandline 
```

Comment:
Could we remove those changes from this PR? This makes the blame quite unreadable.